frontend/src/views/Transfers/__tests__/buttonConfigs.test.jsx

```diff
@@ -95,42 +92,69 @@ describe('buttonClusterConfigFn', () => {
       // signAndSend should now be enabled
       expect(signAndSendButton2.disabled).toBe(false)
     })
-  })
 
-  describe('When status is Draft', () => {
-    it('should return deleteDraft, saveDraft, and signAndSend buttons', () => {
-      // Mock dependencies
-      const toOrgData = []
-      const hasRoles = vi.fn().mockReturnValue(true)
+    it('should disable signAndSend button if user lacks signing authority', () => {
+      const hasRoles = vi.fn().mockReturnValue(false)
       const hasAnyRole = vi.fn().mockReturnValue(true)
       const currentUser = {
         isGovernmentUser: false,
         organization: {
           organizationId: 1
         }
       }
-      const methods = {
-        getValues: vi.fn().mockReturnValue(1)
+      const transferData = null
+
+      signingAuthorityDeclaration = true
+
+      const config = buttonClusterConfigFn({
+        toOrgData,
+        hasRoles,
+        hasAnyRole,
+        currentUser,
+        methods,
+        t,
+        setModalData,
+        createUpdateTransfer,
+        transferData,
+        isGovernmentUser,
+        recommendation,
+        signingAuthorityDeclaration
+      })
+
+      const buttons = config[TRANSFER_STATUSES.NEW]
+
+      const signAndSendButton = buttons.find(
+        (button) => button.id === 'sign-and-send-btn'
+      )
+
+      expect(signAndSendButton.disabled).toBe(true)
+    })
+  })
+
+  describe('When status is Draft', () => {
+    const transferData = {
+      currentStatus: {
+        status: TRANSFER_STATUSES.DRAFT
+      },
+      agreementDate: new Date(),
+      transferHistory: [],
+      fromOrganization: {
+        organizationId: 1
       }
-      const setModalData = vi.fn()
-      const createUpdateTransfer = vi.fn()
-      const transferData = {
-        currentStatus: {
-          status: TRANSFER_STATUSES.DRAFT
-        },
-        agreementDate: new Date(),
-        transferHistory: [],
-        fromOrganization: {
+    }
+
+    it('should return deleteDraft, saveDraft, and signAndSend buttons', () => {
```

Comment:
question: The test mentions it tests for all 3 buttons, but in the asserts I only see a single .toBeDefined(), does one of them need to be updated?

Reply:
Good catch! I added assertions for all three buttons. 👍